.github/workflows/build-and-check.yml

```diff
@@ -99,8 +99,10 @@ jobs:
       - name: set up JDK 1.8
         uses: actions/setup-java@v2
         with:
-          java-version: 8
+          java-version: 11
```

Comment:
might as well run this on java 11

Reply:
Sure... But then change the title of the step too 🙂😂